site/content/docs/0.0/migration.md

```diff
@@ -217,6 +273,13 @@ toc: true
     </ul>
   </details>
 
+- <details class="mb-2">
+    <summary><span class="badge text-bg-danger">Breaking</span> Dropped Sass variables:</summary>
+    <ul>
+      <li><code>$grid-gutter-breakpoint</code></li>
```

Comment:
It's dropped from the Boosted point of view. So this part should rather be in the other migration guide, and not mentioned here as the grids were not supposed to exist before.